xds/internal/xdsclient/xdslbregistry/converter/converter.go

```diff
@@ -152,15 +175,15 @@ func convertWeightedRoundRobinProtoToServiceConfig(rawProto []byte, depth int) (
 	return makeBalancerConfigJSON(weightedroundrobin.Name, lbCfgJSON), nil
 }
 
-func convertV1TypedStructToServiceConfig(rawProto []byte, depth int) (json.RawMessage, error) {
+func convertV1TypedStructToServiceConfig(rawProto []byte, _ int) (json.RawMessage, error) {
 	tsProto := &v1xdsudpatypepb.TypedStruct{}
 	if err := proto.Unmarshal(rawProto, tsProto); err != nil {
 		return nil, fmt.Errorf("failed to unmarshal resource: %v", err)
 	}
 	return convertCustomPolicy(tsProto.GetTypeUrl(), tsProto.GetValue())
 }
 
-func convertV3TypedStructToServiceConfig(rawProto []byte, depth int) (json.RawMessage, error) {
+func convertV3TypedStructToServiceConfig(rawProto []byte, _ int) (json.RawMessage, error) {
```

Comment:
It's not required, but it's a good signal that it's unused.  On the flip side, it can be nice to have the name there in order to document the thing that is unused as well.  🤷